src/signals/IncidentMap/components/GPSLocation/GPSLocation.test.tsx

```diff
@@ -0,0 +1,151 @@
+/* SPDX-License-Identifier: MPL-2.0 */
+/* Copyright (C) 2022 Gemeente Amsterdam */
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import type { Map } from 'leaflet'
+
+import { DEFAULT_ZOOM } from '../../../../components/AreaMap/AreaMap'
+import configuration from '../../../../shared/services/configuration/configuration'
+import { GPSLocation } from './GPSLocation'
+import type { Props } from './GPSLocation'
+
+jest.mock('@amsterdam/react-maps', () => ({
+  __esModule: true,
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  ...jest.requireActual('@amsterdam/react-maps')!,
+  Marker: ({ ...props }) => {
+    return <div {...props}></div>
```

Comment:
You can also just add this: 
``
```suggestion
  ...jest.requireActual('@amsterdam/react-maps'),
  Marker: (props: any) => {
    return <div {...props}></div>
```
You need to add the eslint comment because you've added a `!` at the end. But you don't need that. And spreading the props twice is unnecessary